gb-roms/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "gb-roms"
+version = "0.1.0"
+authors = ["firelight.flagboy <firelight.flagboy@gmail.com>"]
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dev-dependencies]
+# clap-v3 = "3.0.0-beta.1"
+clap = { version = "3.0.0-beta.2" }
+
+[[example]]
```

Comment:
not necessary since a few versions

Reply:
I found a typo on the `examples` folder, now it's working correctly :)